src/irreps/anirrep.jl

```diff
@@ -0,0 +1,304 @@
+"""
+    struct ANIrrep{N} <: AbstractIrrep{Alternating{N}}
+    ANIrrep{N}(n::Integer)
+    Irrep[Alternating{N}](n::Integer)
+
+Represents irreps of the alternating group ``A_N``.
+
+## Fields
+
+- `n::Int`: the label of the irrep.
+
+This can take the value 
+"""
+struct ANIrrep{N} <: AbstractIrrep{Alternating{N}}
+    n::Int
+    function ANIrrep{N}(n::Int) where {N}
+        @assert 0 < N < 5 "ANIrrep is only provided for 0 < N < 5"
```

Comment:
I'm not sure it makes sense to disallow this at the constructor level, as this is just a "to be implemented", not really a fundamental issue?